docs/outlook/onmessagefromchanged-onappointmentfromchanged-events.md

```diff
@@ -53,6 +53,7 @@ The following tables list client-server combinations that support the `OnMessage
 To preview the `OnMessageFromChanged` and `OnAppointmentFromChanged` events, set up your preferred client accordingly.
 
 - For Outlook on Windows, install Version 2212 (Build 15919.10000) or later. Once installed, join the [Office Insider program](https://insider.office.com/join/windows) and select the **Beta Channel** option to access Office beta builds.
+- For Outlook on Mac, install Version 16.69.116 or later. Once installed, join the [Office Insider program](https://insider.office.com/join/Mac) and select the **Beta Channel** option.
```

Comment:
This is in 50% prod already and I have raised request to make in 100% prod. Do you still want to put this message of bet channel switching?

Reply:
@DivyaPatidar Yash advised to keep the beta channel switching guidance until the feature is 100% live. Thanks for raising this.